MAUI/Cartesian-Charts/Trackball.md

```diff
@@ -220,6 +221,189 @@ this.Content = chart;
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart>
+    <chart:SfCartesianChart.Resources>
+        <DataTemplate x:Key="trackballLabelTemplate">
+            <HorizontalStackLayout>
+                <Label Text="{Binding DataItem.Values}" FontSize="20" VerticalOptions="Center" TextColor="White"/>
+                <Image Source="image.png" WidthRequest="40" HeightRequest="40"/>
+            </HorizontalStackLayout>          
+        </DataTemplate>
+    </chart:SfCartesianChart.Resources>
+
+   <chart:SfCartesianChart.TrackballBehavior>
+      <chart:ChartTrackballBehavior/>
+   </chart:SfCartesianChart.TrackballBehavior>
+    . . .
+    <chart:LineSeries ItemsSource="{Binding Data1}" 
+                      TrackballLabelTemplate ="{StaticResource trackballLabelTemplate}"
+                      XBindingPath="Name"
+                      YBindingPath="Values"/>
+
+    <chart:LineSeries ItemsSource="{Binding Data2}" 
+                      TrackballLabelTemplate ="{StaticResource trackballLabelTemplate}"
+                      XBindingPath="Name"
+                      YBindingPath="Values"/>
+
+    <chart:LineSeries ItemsSource="{Binding Data3}" 
+                      TrackballLabelTemplate ="{StaticResource trackballLabelTemplate}"
+                      XBindingPath="Name"
+                      YBindingPath="Values"/>
+                      
+</chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfCartesianChart chart = new SfCartesianChart();
+
+ChartTrackballBehavior trackball = new ChartTrackballBehavior();
+chart.TrackballBehavior= trackball;
+. . .
+LineSeries series1 = new LineSeries();
+series1.ItemsSource = new ViewModel().Data1;
+series1.XBindingPath = "Name";
+series1.YBindingPath = "Values";
+
+LineSeries series2 = new LineSeries();
+series2.ItemsSource = new ViewModel().Data2;
+series2.XBindingPath = "Name";
+series2.YBindingPath = "Values";
+
+LineSeries series3 = new LineSeries();
+series3.ItemsSource = new ViewModel().Data3;
+series3.XBindingPath = "Name";
+series3.YBindingPath = "Values";
+
+DataTemplate trackballLabelTemplate = new DataTemplate(() =>
+{
+    HorizontalStackLayout horizontalStackLayout = new HorizontalStackLayout();
+
+    Label label = new Label
+    {
+        FontSize = 20,
+        VerticalOptions = LayoutOptions.Center,
+        TextColor = Colors.White
+    };
+    label.SetBinding(Label.TextProperty, "DataItem.Values");
+
+    Image image = new Image
+    {
+        Source = "image.png",
+        WidthRequest = 40,
+        HeightRequest = 40
+    };
+
+    horizontalStackLayout.Add(label);
+    horizontalStackLayout.Add(image);
+
+    return horizontalStackLayout;
+});
+
+series1.TrackballLabelTemplate = trackballLabelTemplate;
+series1.TrackballLabelTemplate = trackballLabelTemplate;
```

Comment:
@EzhilarasanElangovan31  series name not updated correctly

Reply:
Updated series name